app/views/offers/new.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Offers#new</h1>
-<p>Find me in app/views/offers/new.html.erb</p>
+<div>
+  <%= simple_form_for(@offer) do |o| %>
```

Comment:
Convention is typically f (for *f*orm), but it's okay

Reply:
ok

Reply:
I will update it. Makes sense

Reply:
done